src/main/java/com/scalyr/log4j/ScalyrAppender.java

```diff
@@ -2,41 +2,64 @@
 
 import com.scalyr.api.logs.EventAttributes;
 import com.scalyr.api.logs.Events;
+import com.scalyr.util.Util;
 import org.apache.log4j.AppenderSkeleton;
 import org.apache.log4j.Level;
 import org.apache.log4j.spi.LoggingEvent;
 
+import java.util.Map;
+
 public class ScalyrAppender extends AppenderSkeleton {
     private String apiKey;
     private String serverHost = "";
+    private String env = "";
     private String logfile = "log4j";
     private String parser = "log4j";
+    private String extraAttributes = "[]";
```

Comment:
Should be "", not "[]", correct?

Reply:
good catch. that was from a previous version where i was using json instead of kv pairs... scrapped it because i didn't want to pull in another runtime dependency to parse json.